interface/templates/interface/submission_result.html

```diff
@@ -6,19 +6,35 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
         <!-- Bootstrap CSS -->
-        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://unpkg.com/bootstrap@4.3.1/dist/css/bootstrap.min.css">
+
+        <!-- Holly Javascript -->
+        <script src="http://unpkg.com/jquery@3.2.1/dist/jquery.js"></script>
+        <script src="https://unpkg.com/bootstrap@4.3.1/dist/js/bootstrap.js"></script>
 
         <title>VMCK</title>
+
+        {% include "interface/review_modal.html" %}
```

Comment:
I'm pretty sure this is not supposed to be within `<head>`. A better place would be at the very bottom of the page, before `</body>`.